www/src/components/package-readme.js

```diff
@@ -40,9 +42,8 @@ const PackageReadMe = props => {
           }}
           href={githubUrl}
           aria-label={`${packageName} source`}
-          title={`View source on GitHub`}
         >
-          <GithubIcon focusable="false" style={{ verticalAlign: `text-top` }} />
+          <GithubIcon focusable="false" style={{ verticalAlign: `text-top` }} />&nbsp;&nbsp;<span>View plugin on GitHub</span>
```

Comment:
Can you remove the `&nbsp;&nbsp;` and instead add the spacing with CSS?

Reply:
Yep, no problem!